### PR TITLE
some change in footer

### PR DESCRIPTION
**Pull request by tanktejas**
=> added Twitter and youtube icons in the footer.
=> add email in the footer issue #716

### CODE REVIEW

File: frontend/src/components/Footer/footer.module.scss [hunk 1]
```diff
@@ -92,13 +92,16 @@
   font-size: 15px;
   font-weight: 400;
 }
-
+.footer_footer-addr__pYXWE {
```
Comment (Kajol-Kumari) [inline frontend/src/components/Footer/footer.module.scss:95]: please remove this as it's not required

File: frontend/src/components/Footer/footer.module.scss [hunk 2]
```diff
@@ -92,13 +92,16 @@
...
+}
 .nav-title {
   font-weight: 600;
   font-size: 18px;
   color: #fff;
 }
-
+.footer div {
```
Comment (Kajol-Kumari) [inline frontend/src/components/Footer/footer.module.scss:102]: please leave one line blank before starting a new class

File: frontend/src/components/Footer/footer.module.scss [hunk 3]
```diff
@@ -92,13 +92,16 @@
...
+  margin-bottom: 1em !important;
+}
```
Comment (Kajol-Kumari) [inline frontend/src/components/Footer/footer.module.scss:104]: please leave one line blank before starting a new class

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@tanktejas please do the following changes -
1. The minor enhacements suggetsed
2. Replace `email` icon with `instagram` icom

**Review by AyushSingh22 — COMMENTED** [review]
Any Update?? @tanktejas

**Comment by harshcodesit** [comment]
My name is Harshwardhan Singh Panwar and I'm a contributor at GSSOC'24 .

idea:- By incorporating div CSS and animations, I can design the UI in an interactive manner, making it visually appealing.

Please assign this task to me, GSSOC'24 .

**Pull request closed without merging**